src/util/profile/profile.ts

```diff
@@ -83,7 +84,7 @@ class ProfileImpl implements Profile {
     return this;
   }
 
-  setAccessToken(token: TokenValueType): Promise<Profile> {
+  setAccessToken(token: TokenValueType): Promise<void> {
```

Comment:
The actual Promise returns `this` so I updated the function definition to match it